qpid-docker/README.md

```diff
@@ -0,0 +1,234 @@
+## Docker Image Example
+
+This is an example on how a Docker Image For Apache Qpid Broker-J based on Eclipse Temurin JRE image can be created.
+
+## Building Container Image
+
+To use an official Apache release in your image run the following command from the qpid-docker directory where
+<QPID_RELEASE_VERSION> is the release version you wish to use (e.g. 9.1.0):
+
+```
+cd qpid-docker
+
+docker-build.sh --release <QPID_RELEASE_VERSION>
+```
+
+This will download the Qpid Broker-J release and copy all the files necessary to build the pre-configured Docker image
+and provide you with additional instructions. Follow these instructions to finish building the image you want based on
+the provided Docker file or even one of your own.
+
+If you would rather prefer to build the docker image from local Broker-J distribution, build the parent project using
+the command
+
+```
+mvn clean install -DskipTests=true
+```
+
+Navigate to the module 'qpid-docker':
+
+```
+cd qpid-docker
+```
+
+Execute the command
+
+```
+docker-build.sh --local-dist-path <PATH_TO_LOCAL_QPID_DISTRIBUTION>
+```
+
+This will copy all the files necessary to build the pre-configured Docker image and provide you with additional
+instructions. Follow these instructions to finish building the image you want based on one of the provided Docker file
+or even one of your own.
+
+### Container Structure
+
+Broker-J files are copied to the folder /qpid-broker-j \
+This folder belongs to user qpid, which is part of the root group. Java process is executed under the qpid user as well.
+
+### Running the Container
```

Comment:
Might be worth adding a line somewhere about stopping the started container for completeness.